tests/Feature/ResetPasswordTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Services\ResetPasswordService;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use App\Services\UserService;
+use Tests\TestCase;
+
+class ResetPasswordTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_if_user_gets_reset_token()
+    {
+        $this->artisan('passport:install', ['--force' => true]);
+
+        $userService = app()->make(UserService::class);
```

Comment:
laravel factory to create test user 

https://laravel.com/docs/9.x/database-testing#model-factories